### PR TITLE
Add graphviz package to kubekins image for graphing profiles

### PR DESCRIPTION
**Pull request by shyamjvs**
/cc @wojtek-t

### CODE REVIEW

**Comment by shyamjvs** [comment]
fyi - @krzyzacy @BenTheElder

**Comment by shyamjvs** [comment]
Ref umbrella issue - https://github.com/kubernetes/kubernetes/issues/59760

File: images/kubekins-e2e/Dockerfile [hunk 1]
```diff
@@ -58,6 +58,9 @@ ENV IMAGE=${IMAGE_ARG}
 # preinstall this for kops tests xref kubetest prepareAws(...)
 RUN pip install awscli
 
+# preinstall graphviz package for graphing profiles
+RUN apt-get install -y graphviz
```
Comment (BenTheElder) [inline images/kubekins-e2e/Dockerfile]: This should not be another step.

**Comment by krzyzacy** [comment]
why we need this in kubekins image?

**Comment by shyamjvs** [comment]
@krzyzacy It's an important tool for creating profiles, which we need for measuring k8s performance. Size-wise it is very much (the kubekins-e2e image earlier was ~615 MB, and now it is ~625 MB). Ref - https://pantheon.corp.google.com/gcr/images/k8s-testimages/GLOBAL/kubekins-e2e?gcrImageListsize=50 (access-controlled link).

File: images/kubekins-e2e/Dockerfile (hunk 1 above)
Comment (shyamjvs) [reply images/kubekins-e2e/Dockerfile]: Could you clarify? Do you mean I should add it to some already existing `RUN` command above?

**Comment by shyamjvs** [comment]
Also, [profile-gathering library](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/profile_gatherer.go) is now part of our e2e testing framework (and can potentially be called from any e2e test). So IMO we should have this available as part of our `kubekins-e2e` image. Wdyt?

File: images/kubekins-e2e/Dockerfile (hunk 1 above)
Comment (BenTheElder) [reply images/kubekins-e2e/Dockerfile]: This image has an unfortunately long build and number of layers, I thought we still had another `apt-get` call here.

This probably make no difference, I withdraw. I do wonder if this should be in the image though.

**Pull request merged**